src/Perlang.Interpreter/Compiler/PerlangCompiler.cs

```diff
@@ -848,14 +848,6 @@ private void AddGlobalClass(string name, PerlangClass perlangClass)
                     // Normal math.h-based pow(), returning a double
                     currentMethod.Append($"{leftCast}pow({expr.Left.Accept(this)}, {rightCast}{expr.Right.Accept(this)})");
                 }
-                else if (expr.Left.TypeReference.ClrType == typeof(BigInteger) ||
-                    expr.Right.TypeReference.ClrType == typeof(BigInteger))
-                {
-                    // TODO: We need something lke BigInteger.Pow() in .NET to be able to accomplish this. Perhaps do
-                    // TODO: like the .NET folks and implement something using this approach?
-                    // TODO: https://en.wikipedia.org/wiki/Exponentiation_by_squaring
-                    throw new NotImplementedInCompiledModeException("** operator for BigInteger is not yet supported in compiled mode");
-                }
```

Comment:
Don't know what I was thinking when I added this. :see_no_evil: `BigInt ** BigInt` is not even supported in interpreted mode..